harry-core/src/harry/data/ResultSetRow.java

```diff
@@ -55,12 +78,26 @@ public ResultSetRow clone()
                                 Arrays.copyOf(vds, vds.length), Arrays.copyOf(lts, lts.length));
     }
 
+    public boolean equals(Object o)
```

Comment:
nit: `@Override` ?

I don't know if you'll need a `hashCode()` here or ever have this in a collection, but I'm compelled by habit to mention it :p